src/HttpRequest.cpp

```diff
@@ -74,9 +76,19 @@ std::string 				HttpRequest::translateMethod(HttpRequest::METHOD method) const
 		case HttpRequest::METHOD::CGI_POST :
 			return HTTP_POST;
 		break;
-		
+
 		default:
 		break;
 	}
 	return nullptr;
-}
\ No newline at end of file
+}
+
+void HttpRequest::setParameters(std::string &uri)
+{
+	std::cout << "Uri parameters " << uri << std::endl;
```

Comment:
I guess this was just for debugging?

Reply:
yeah, actually added the parameters to the httprequest in the next commit :P